packages-exp/analytics-exp/src/api.ts

```diff
@@ -57,15 +58,38 @@ declare module '@firebase/component' {
   }
 }
 
+/**
+ * Firebase Analytics initialization function that allows custom
+ * settings. Can be called at most once. If no custom settings
```

Comment:
Should it throw if it's called more than once, or after getAnalytics() is called for the same app instance?

Also, `settings()` manages a global state that should not change from instance to instance. Does it even make sense to call it as part of `initializeAnalytics()` which is creating an instance. It seems to make more sense to let people call `settings()` manually once before calling any `getAnalytics()`. So perhaps we don't need `initializeAnalytics()` ?

Reply:
Changed PR to just remove analytics prefix from analyticsSettings.